admob/src/android/banner_view_internal_android.cc

```diff
@@ -390,21 +386,29 @@ Future<void> BannerViewInternalAndroid::Resume() {
 
 Future<void> BannerViewInternalAndroid::Destroy() {
   firebase::MutexLock lock(mutex_);
-  return InvokeNullary(kBannerViewFnDestroy, banner_view_helper::kDestroy);
+  FutureCallbackData<void>* callback_data =
```

Comment:
No longer use futures to track the destruction as they may be destroyed before the're completed. This had caused a crash on some Android runs.

TBH, the Applicatoin _should already have invoked_ `Destroy()` prior to this object's destruction.  Invoking it again here isn't required.  I guess it was put here to protect our users from accidentally leaking memory. 

There doesn't seem to be a good reason to block the destructor though.  Instead I'll start an asynchronous attempt to destroy the Java resources, if they haven't been destroyed already, and then return immediately here.

Reply:
Yeah, I think there's no reason to actually wait for the Java destruction to complete.